### DISCUSSION TITLE
How to add this parameters X-Confirm-Reading-To: Disposition-Notification-To: or Return-Receipt-To:

### DISCUSSION THREAD

**Discussion by vecchiojoe** [PHPMailer/PHPMailer#3058] [General]
Hi, i'm tryng to add into my class this params
X-Confirm-Reading-To: 
Disposition-Notification-To: 
Return-Receipt-To:
But seem not working
This a part of the code where i added the params
![image](https://github.com/PHPMailer/PHPMailer/assets/2722518/a7b33b48-bf08-41fa-854e-6c60e8b1bbc4)

Thanks very much for you help
Joe

**Comment by Synchro** [comment]
That looks correct. Can you see the headers in the sent message if you set SMTPDebug=2?

**Reply by vecchiojoe** [reply]
Hi, thank for the answer
no that params not showing

**Reply by vecchiojoe** [reply]
i did, but no error message
email arriving in outlook

**Reply by Synchro** [reply]
You're not checking the return values from the calls to addCustomHeader - they should return true if successful: check that in your code.
